openmsupply/open-msupply
# 1726 Outbound Shipment: can "info" message disappear once shipment is confirmed as "Delivered"

Fixes #1726

# 👩🏻‍💻 What does this PR do? 
Have two info messages for automatically created shipments and load them by the inbound shipment status.

# 🧪 How has/should this change been tested? 
- Send an `Outbound Shipment` to any store
- Go to that store and go to `Inbound Shipment`
- Click on the invoice: see full info message
- Change status of invoice to `Delivered`: see only first half of the info message
